visidata/features/regex.py

```diff
@@ -120,11 +124,15 @@ def regex_flags(sheet):
     'Return flags to pass to regex functions from options'
     return sum(getattr(re, f.upper()) for f in sheet.options.regex_flags)
 
+@BaseSheet.api
+def verify_regex_capture(sheet, regexstr):
```

Comment:
Let's move this check to the existing `makeRegexMatcher` (outside the wrapper).

Reply:
That ended up making it a lot easier!